Document screenshots as common content (#4920)
getsentry/sentry-docs: src/platforms/common/enriching-events/screenshots.mdx

---
title: "Screenshots"
sidebar_order: 106
description: "Learn more about taking screenshots when an error occurs. Sentry pairs the screenshot with the original event, giving you additional insight into issues."
supported:
  - android
---

<Note>

The Screenshots feature is available in Beta. Beta features are still in-progress and may have bugs. We recognize the irony.

</Note>

> When a user experiences an error, Sentry provides the ability to take a screenshot and include as an [attachment](/platforms/android/enriching-events/attachments/).

Author: @imatwawana is there a way to link to `enriching-events/attachments` page but not from the `android` platform? rather redirecting to the current platform? this content will be available for other platforms as well.

Reviewer: A PlatformLink component should work: 

<PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>

> When a user experiences an error, Sentry provides the ability to take a screenshot and include as an [attachment](/platforms/android/enriching-events/attachments/).

Reviewer: ```suggestion
When a user experiences an error, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
```

> This feature only applies to SDKs with a user interface, such as Mobile and Desktop. In some environments such as native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently this feature will be a best effort solution.

Reviewer: ```suggestion
This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. So inherently, this feature will be a best effort solution.
```

## Enabling Screenshots

> Screenshots may contain [PII](/platforms/android/data-management/sensitive-data/) and is an opt-in feature, you can enable that by doing.

Reviewer: ```suggestion
Screenshots may contain [PII](/platforms/android/data-management/sensitive-data/) and is an opt-in feature. You can enable it as shown below:
```

> Screenshots may contain [PII](/platforms/android/data-management/sensitive-data/) and is an opt-in feature. You can enable it as shown below:

Author: ```suggestion
Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink>[PII] and is an opt-in feature. You can enable it as shown below:
```

<PlatformContent includePath="enriching-events/attach-screenshots" />

## Viewing Screenshots

> Screenshots appear on the **Issue Details** page, where you can view the thumbnail, as well as associated events. Click the Event ID to open the **Issue Details** of that specific event.

Reviewer: ```suggestion
Screenshots appear on the **Issue Details** page, where you can view the thumbnail, as well as associated events. Click the event ID to open the **Issue Details** of that specific event.
```

![Screenshots Thumbnail Example](screenshots/screenshot-thumbnail.png)

> Alternately, screenshots also appear in the _Attachments_ tab on the **Issue Details** page, where you can view all attachments, as well as associated events. Click the Event ID to open the **Issue Details** of that specific event.

Reviewer: ```suggestion
Alternatively, screenshots are also available in the "Attachments" tab on the **Issue Details** page, where you can view all attachments, as well as associated events. Click the event ID to open the **Issue Details** of that specific event.
```
